Sigh... DAutoTest:
```
.generated/stable_dmd-2.088.0/dmd2/linux/bin64/dub build --compiler=.generated/stable_dmd-2.088.0/dmd2/linux/bin64/dmd --root=ddoc && \
	mv ddoc/ddoc_preprocessor .generated/ddoc_preprocessor
dub-1.17.0 does not comply with toolchainRequirements.dub specification: >=1.29.0
Please consider upgrading your DUB installation
make: *** [posix.mak:919: .generated/ddoc_preprocessor] Error 2
```

--- a/.github/workflows/build_with_dub.yml
+++ b/.github/workflows/build_with_dub.yml
@@ -17,5 +17,8 @@ jobs:
             - name: Install Host D compiler
               uses: dlang-community/setup-dlang@v1
 
-            - name: Invoke dub
+            - name: Build and run compiler
               run: dub run dmd:compiler -- --version
+
+            - name: Build frontend library
+              run: dub build dmd:frontend

